common/inc/sgx_error.h

```diff
@@ -61,7 +61,7 @@ typedef enum _status_t
     SGX_ERROR_NDEBUG_ENCLAVE     = SGX_MK_ERROR(0x2004),      /* The enclave is signed as product enclave, and can not be created as debuggable enclave. */
     SGX_ERROR_OUT_OF_EPC         = SGX_MK_ERROR(0x2005),      /* Not enough EPC is available to load the enclave */
     SGX_ERROR_NO_DEVICE          = SGX_MK_ERROR(0x2006),      /* Can't open SGX device */
-    SGX_ERROR_MEMORY_MAP_CONFLICT= SGX_MK_ERROR(0x2007),      /* Page mapping failed in driver */
+    SGX_ERROR_MEMORY_MAP_CONFLICT= SGX_MK_ERROR(0x2007),      /* Page mapping failed in driver. Deprecated, replaced with SGX_ERROR_MEMORY_MAP_FAILURE*/
```

Comment:
the relationship for SGX_ERROR_MEMORY_MAP_CONFLICT and SGX_ERROR_MEMORY_MAP_FAILURE is wrong
don't modified the comments here.

Reply:
![image](https://user-images.githubusercontent.com/39938149/159613447-36882235-edb0-42f2-bc98-21883a877618.png)

So, this is not right?

Reply:
SGX_ERROR_MEMORY_MAP_CONFLICT is not to replace SGX_ERROR_MEMORY_MAP_FAILURE.
So the comment is not right here.

Reply:
agreed, please remove the part saying "replaced with ...", but still mark it as deprecated because we do not have it returned from lower level code anymore.

Reply:
ok, i fix my code. please check it.